### PR TITLE
Fixed permissions with files

### PR DESCRIPTION
**Pull request by bio-boris**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -48,10 +29,28 @@ RUN apt-get install -y apt-transport-https software-properties-common && \
...
     usermod -a -G 999 kbase
 
-USER kbase:999
+# The htcondor package tries an interactive config, set DEBIAN_FRONTEND to
+# noninteractive in order to prevent that
+RUN export DEBIAN_FRONTEND=noninteractive && \
+    apt-get install -y htcondor zile vim libgomp1 && \
+    mkdir /scratch && \
+    cd /tmp && \
+    wget http://submit-3.batlab.org/nmi-runs/condorauto/2018/03/condorauto_submit-3.batlab.org_1520871025_1539075/userdir/nmi:x86_64_Debian9/results.tar.gz && \
```
Comment (kkellerlbl) [inline Dockerfile]: What is this?

File: deployment/conf/.templates/start_server.sh.templ [hunk 2]
```diff
@@ -7,6 +7,8 @@ if [ "$POOL_PASSWORD" ] ; then
     /usr/sbin/condor_store_cred -p "$POOL_PASSWORD" -f `condor_config_val SEC_PASSWORD_FILE`
 fi
 
+chown kbase `condor_config_val SEC_PASSWORD_FILE`
```
Comment (kkellerlbl) [inline deployment/conf/.templates/start_server.sh.templ]: Are the daemons running as root or as kbase?  I thought we switched to root.

File: Dockerfile (hunk 1 above)
Comment (bio-boris) [reply Dockerfile]: Not sure, this was already in the debian image that Steve created

Comment (bio-boris) [reply Dockerfile]: Also I separated out the steps for easier debugging

File: deployment/conf/.templates/start_server.sh.templ (hunk 2 above)
Comment (bio-boris) [reply deployment/conf/.templates/start_server.sh.templ]: This is not for the daemons. This is for starting the java server. However, the KBASE user needs to be able to read that password file.

Comment (bio-boris) [reply deployment/conf/.templates/start_server.sh.templ]: @kkellerlbl

**Pull request merged**